.fixtures.yml

```diff
@@ -1,6 +1,12 @@
 fixtures:
   repositories:
     apt: "git://github.com/puppetlabs/puppetlabs-apt.git"
+    cron_core:
+      repo: "https://github.com/puppetlabs/puppetlabs-cron_core"
+      puppet_version: '>= 6.0.0'
     stdlib: "git://github.com/puppetlabs/puppetlabs-stdlib.git"
+    yumrepo_core:
```

Comment:
do we need this module if we only support Debian based instances ? 🤔

Reply:
Since the module use `yumrepo` in `manifests/repository/yum.pp`, and `yumrepo` is no longer a built-in module, I had to add this too.  Same remark as above: puppet-agent ships with the module, so there is no need to update `metadata.json`.